e2e/test/e2e/ui/git_integrations_test.exs

```diff
@@ -0,0 +1,164 @@
+defmodule E2E.UI.GitIntegrationsTest do
+  use E2E.UI.UserTestCase
+
+  def navigate_to_git_integrations(session, organization, base_domain) do
+    git_integrations_url = "https://#{organization}.#{base_domain}/settings/git_integrations/"
+    Logger.info("Navigating to Git Integrations page: #{git_integrations_url}")
+
+    session
+    |> visit(git_integrations_url)
+  end
+
+  describe "Git Integrations page" do
+    setup %{session: session, organization: organization, base_domain: base_domain} do
+      session = navigate_to_git_integrations(session, organization, base_domain)
+      {:ok, %{session: session}}
+    end
+
+    test "has correct title and description", %{session: session} do
+      session
+      |> assert_has(Wallaby.Query.text("Git Integrations", count: 1))
+    end
+
+    test "has two main sections: Integrations and Connect new", %{session: session} do
+      session
+      |> assert_has(
+        Wallaby.Query.css(".bb.b--black-075.w-100-l.mb4.br3.shadow-3.bg-white",
+          count: 2
+        )
+      )
+      |> assert_has(
+        Wallaby.Query.css(".bb.bw1.b--black-075.br3.br--top .flex.items-center .b",
```

Comment:
Can't we use an ID on the element? If we assert on these CSS classes, every time we change something on the CSS, we'll have to update this test